code/game/gamemodes/wizard/artefact.dm

```diff
@@ -708,6 +708,32 @@ var/global/list/multiverse = list()
 			H.equip_to_slot_or_del(new /obj/item/weapon/shield/riot/roman(H), slot_l_hand)
 			H.equip_to_slot_or_del(new /obj/item/weapon/twohanded/spear(H), slot_back)
 
+/obj/item/device/necromantic_stone/proc/spawnheresy(mob/living/carbon/human/H as mob)
+	H.set_species("Human")
+	if (H.gender == MALE)
+		H.gender = FEMALE
+	H.h_style = random_hair_style(H.gender, H.species)
```

Comment:
Maybe if you don't want them bald you should set H.h_style to the floorlength braid or something edgy and anime-like